riscv-debug-spec.tex

```diff
@@ -619,6 +642,34 @@ \subsection{JTAG Connector}
 connector looks as it does in Table~\ref{tab:header}. The function of each pin
 is described in Table~\ref{tab:pinout}.
 
+%MAW: Why do we only have 'RESET', but no 'TRST(n)' signal?
+% See http://openocd.org/doc/html/Reset-Configuration.html
+% Both are useful.
+% One is explicitly ``test logic reset'', which would imply
+% resetting the JTAG state machine and all associated test logic, in this
+% case the Debug Transport Module, synchronizers, and perhaps even
+% the Debug Module itself. It basically does the same thing as getting
+% the JTAG state machine into TEST_LOGIC_RESET state, which is
+% why it is optional.
+%
+% The other is a convenience which allows the debugger to essentially
+% hit the system reset button, which is basically mutually exclusive from
+% what TRST resets.
```

Comment:
Pins are expensive, and this one doesn't seem essential.
We copied the Cortex Debug Connector because it's compact and few pins. It doesn't have a TAP reset signal either. 1149.1 explicitly says TRST is an optional signal.

> TRST* is required when the test logic does not power-up in a known and controlled state, either because of the nature of the underlying technology or because there is no on-chip power-up reset generator.

That implies POR is fine for those circuits.